lib/index.js

```diff
@@ -169,24 +169,24 @@ FSTree.prototype = {
   },
 
   get _relativePathToChange() {
-    return this._parent ? this._parent._relativePathToChange : this.__relativePathToChange;
+    return this.parent ? this.parent._relativePathToChange : this.__relativePathToChange;
   },
 
   set _relativePathToChange(value) {
-    if (this._parent) {
-      this._parent._relativePathToChange = value;
+    if (this.parent) {
+      this.parent._relativePathToChange = value;
```

Comment:
Seems good although we'll have to do a think-through and make sure that we're not trolling ourselves with `projectionTree.start();`